next.config.js

```diff
@@ -59,11 +59,16 @@ const nextConfig = {
               `connect-src 'self' ${
                 process.env.NODE_ENV === "development" ? "webpack://*" : ""
               } https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com https://*.ingest.sentry.io`,
-              `img-src 'self' https://*.google-analytics.com https://*.googletagmanager.com https://firefoxusercontent.com https://mozillausercontent.com https://monitor.cdn.mozilla.net ${
-                process.env.FXA_ENABLED
-                  ? new URL(process.env.OAUTH_PROFILE_URI).origin
```

Comment:
This should have already resulted in the profile image being allowlisted on stage, but I think these environment variables just aren't set to what I thought they would be there. Either way, since we're already hardcoding them in an allowlist above, I figured it made sense to re-use that, in case new URLs get added there later.